docs/resources/ocean_aks_np.md

```diff
@@ -276,3 +276,35 @@ The following arguments are supported:
     * `min_nics` - (Optional) Minimum number of network interfaces.
     * `min_disk` - (Optional) Minimum number of data disks available.
     * `vm_types` - (Optional, Enum `"generalPurpose", "memoryOptimized", "computeOptimized", "highPerformanceCompute", "storageOptimized", "GPU"`) The filtered vm types will belong to one of the vm types from this list.
+
+<a id="update-policy"></a>
+## Update Policy
+
+* `update_policy` - (Optional)
+  * `should_roll` - (Required) Enables the roll.
```

Comment:
add - if set to true along with the cluster update, roll will be triggered

Reply:
Added